### ISSUE TITLE
A11Y: Increase toast timeout

### ISSUE THREAD

**Issue by alexstine** [bbertucc/equalify#463]
The toast notifications should have a higher time limit before being removed from the DOM.

https://www.w3.org/WAI/WCAG22/Understanding/timing-adjustable.html

I argue that the current timeout is a partial violation of WCAG 2.2.1: Timing Adjustable. Even though the user is not completing an activity, if the user misses the aria-live announcement and needs to read the message again, they will not have enough time to do so as the content is removed from the DOM.

> The intent of this Success Criterion is to ensure that users with disabilities are given adequate time to interact with Web content whenever possible. People with disabilities such as blindness, low vision, dexterity impairments, and cognitive limitations may require more time to read content or to perform functions such as filling out on-line forms.

In this example, reading content of the toast.

**Comment by bbertucc** [comment]
Thanks @alexstine adding this to our current sprint so we discuss it at our Monday contributor meeting.

**Comment by bbertucc** [comment]
Discussed this on our contributor call today. @wilsuriel03 is going to post the commit that changed the timeout settings.

**Comment by bbertucc** [comment]
@wilsuriel03 any updates with this? You said you would go into past commits.
